From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: John Kleinschmidt <jkleinsc@github.com>
Date: Tue, 28 Jan 2020 12:12:28 -0500
Subject: Remove serialization/deserialization of WasmModuleObject

This is already present in Node.js v14 and can be removed when we upgrade.

Serialization of WasmModuleObject was removed here:
https://chromium.googlesource.com/v8/v8/+/30e4ba6df4cdf5582de4d79850bcd270e6a75a7a

diff --git a/test/parallel/test-v8-serdes.js b/test/parallel/test-v8-serdes.js
index a992ba42ce46bfcdccdca997ee13718086d3256d..62096edec540ec3dd70e56fee53b9813b2ebbbfc 100644
--- a/test/parallel/test-v8-serdes.js
+++ b/test/parallel/test-v8-serdes.js
@@ -23,8 +23,7 @@ const objects = [
   undefined,
   null,
   42,
-  circular,
-  wasmModule
+  circular
 ];
 
 const hostObject = new (internalBinding('js_stream').JSStream)();
@@ -236,9 +235,3 @@ const deserializerTypeError =
     /^TypeError: buffer must be a TypedArray or a DataView$/,
   );
 }
-
-{
-  const deserializedWasmModule = v8.deserialize(v8.serialize(wasmModule));
-  const instance = new WebAssembly.Instance(deserializedWasmModule);
-  assert.strictEqual(instance.exports.add(10, 20), 30);
-}
